lib/rawQueries.js

```diff
@@ -9,12 +9,14 @@ const BuildFactoryQueries = {
         WHERE rank > :offset AND rank <= :maxRank
         ORDER BY "jobId", "id" ASC`,
 
-    statusesQueryMySql: `SELECT id, jobId, status, startTime, endTime
-        FROM (SELECT id, jobId, status, startTime, endTime,
-        RANK() OVER ( PARTITION BY jobId ORDER BY id DESC ) AS \`rank\`
-        FROM builds WHERE jobId in (:jobIds)) as R
-        WHERE \`rank\` > :offset AND \`rank\` <= :maxRank
-        ORDER BY jobId, id ASC`,
+    statusesQueryMySql: `SELECT id, jobId, status, startTime, endTime FROM (
+	    SELECT a.id, a.jobId, a.status, a.startTime,  a.endTime, count(b.id)+1 AS rank
+		FROM  builds a LEFT JOIN (SELECT id, jobId FROM builds WHERE jobId IN (:jobIds)) b
+                ON a.id>b.id AND a.jobId=b.jobId
```

Comment:
The postgresql one is ranking by `id DESC`, which mean the latest build comes first in ranking. Should this be `a.id < b.id` instead so the latest build from `a` will have the rank of `1` cuz it should have the least count?

Reply:
did this based on current api output in postgres - https://github.com/screwdriver-cd/models/pull/456#issuecomment-638590892

Reply:
you are correct, updated query